swaglyrics/unsupported.txt

```diff
@@ -238,3 +238,5 @@ Out Of My Mind - R3HAB Edit by Skytech
 Wherever You Go by Tim Schaufert
```

Comment:
You can just leave this file empty. The songs will update automatically on first run 👍

Reply:
Hasn't been resolved.